### ISSUE TITLE
leaflet-geocoder-mapzen.css missing

### ISSUE THREAD

**Issue by leezu** [pelias/leaflet-plugin#140]
With `leaflet-geocoder-mapzen@1.7.0` the `leaflet-geocoder-mapzen.css` file is not present in the dist directory after installing. The .js file is properly copied however.

I declared leaflet-geocoder as a dependency of my software as follows:

```
"leaflet-geocoder-mapzen": "^1.6.3"
```

The dist folder looks as follows:

``` node_modules/leaflet-geocoder-mapzen/dist/
├── images
│   ├── loading@2x.gif
│   ├── loading.gif
│   ├── point_icon@2x.png
│   ├── point_icon.png
│   ├── polygon_icon@2x.png
│   ├── polygon_icon.png
│   ├── search@2x.png
│   └── search.png
└── leaflet-geocoder-mapzen.js
```

**Comment by louh** [comment]
Indeed it is. We changed the deploy process and things borked. I'm looking into it today.

**Comment by louh** [comment]
@leezu We've deprecated 1.7.0 and published 1.7.1 which should put the CSS (and minified files) back. Please try that and let me know if that works.

**Comment by leezu** [comment]
Works fine now, thanks!
